randovania/games/am2r/json_data/The Tower.txt

```diff
@@ -211,20 +215,28 @@ Extra - map_name: rm_a4h03
   > Pickup (Missile Tank)
       All of the following:
           Morph Ball
-          Power Grip or Mid-Air Morph (Intermediate) or Can Use Spider Ball
           Any of the following:
+              # Break the bomb blocks
               Can Use Power Bombs
               All of the following:
                   Can Use Bombs
-                  Infinite Bomb Jumping (Intermediate) or Can Use Spider Ball
+                  Infinite Bomb Jumping (Advanced) or Can Use Spider Ball
           Any of the following:
-              Space Jump or Walljump (Advanced)
-              Speed Booster and Shinesparking Tricks (Beginner)
+              # Getting up to the tunnel
+              Can Use Spider Ball
+              All of the following:
+                  # Space Jump Climb
+                  Space Jump
+                  Power Grip or Mid-Air Morph (Intermediate)
               All of the following:
-                  Hi-Jump Boots
+                  # Shinespark Method
+                  Speed Booster and Shinesparking Tricks (Intermediate)
                   Any of the following:
-                      Mid-Air Morph (Intermediate) or Walljump (Intermediate)
-                      Power Grip and Movement (Advanced)
+                      Power Grip
+                      Mid-Air Morph (Advanced) and Walljump (Advanced)
+              All of the following:
+                  # IBJ
```

Comment:
would be nice if this IBJ had "from midair or very close edge" or something commented